bridges/otelzap/core_test.go

```diff
@@ -29,16 +29,61 @@ func TestCore(t *testing.T) {
 	zc := NewCore(loggerName, WithLoggerProvider(rec))
 	logger := zap.New(zc)
 
-	logger.Info(testMessage, zap.String(testKey, testValue))
+	t.Run("Test 'Write' method", func(t *testing.T) {
```

Comment:
This and the following test has the same name. Can we make this more descriptive of what is being tested?